src/lib/radio/_radio-theme.scss

```diff
@@ -9,33 +9,57 @@
   $background: map-get($theme, background);
   $foreground: map-get($theme, foreground);
 
-  .mat-radio-outer-circle {
-    border-color: mat-color($foreground, secondary-text);
-  }
+    .mat-radio-button {
+        //default color (accent)
+        &:not(.mat-primary):not(.mat-accent):not(.mat-warn) {
```

Comment:
I don't think this `:not` expression should be necessary; the radio should just have `accent` be the default color.